htdocs/admin/dict.php

```diff
@@ -1775,7 +1775,7 @@
 				}
 
 				if (empty($reshook)) {
-					fieldList($fieldlist, $obj, $tabname[$id], 'add');
+					dict_fieldList($fieldlist, $obj, $tabname[$id], 'add');
```

Comment:
If we rename a function, we must use a camelcase format function name
For example dictFieldList()

Reply:
@eldy Good to know, I changed to camelcase.